src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const partnerType = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
it's a good practice to use object destructing (use destructing elements below as well)
```suggestion
export const Person = ({ person }) => {
  const {...} = person;
  const partnerType = sex === 'm' ? 'wife' : 'husband';
```